chipcie/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
The chart will mainly host static files if I understand correctly. I would say that testing the connection to those is unnecessary

Reply:
fixed, this test is helm specific and came out of the box, it will use this as a test for helm to check if an upgrade is successful. I added an rolling update strategy in the deployment which basically let k8s do it, since updates are not propagated by a helm chart update, but a change to the image version number.

Reply:
Great!